webapp/templates/jury/jury_macros.twig

```diff
@@ -141,6 +141,10 @@
                             {{- (item.value|default(item.default|default('')))|affiliationLogo(item.title) -}}
                             {% elseif key == "warning_content" %}
                             {{- item.value|printWarningContent -}}
+                            {% elseif key == "badges" %}
+                                {% for badge in item.value %}
+                                    {{- badge|problemBadge -}}
```

Comment:
Maybe we want a space between the badges?

Reply:
![image](https://github.com/DOMjudge/domjudge/assets/14887731/760af524-543a-480c-a2e8-9c50867bf2d5)

Fine by me.